content/01.abstract.md

```diff
@@ -1,10 +1,14 @@
 ## Abstract {.page_break_before}
 
-Important tasks in biology and biomedicine such as predicting gene functions, gene-disease associations, and drug repurposing opportunities are often framed as network edge prediction.
-Degree strongly influences edge prediction, and degree biases and degree imbalance can lead to nonspecific predictions.
-We introduce a network permutation framework to quantify the effect of node degree on network-based methods and prediction tasks.
-As part of this we define an edge prior to quantify the probability that two nodes are connected based only on their degree.
-After demonstrating that this prior feature shows excellent discrimination and calibration performance for 20 different biomedical networks (16 bipartite, 3 undirected, 1 directed), we conclude that our prior feature represents a suitable baseline for network link prediction tasks, as performance exceeding the baseline is attributable to factors other than degree alone.
-Additionally, we propose methods to incorporate network permutation and the edge prior into other predictive methods.
-Our results highlight the importance of degree for link prediction and provide software to account for its effects when degree bias may be present.
-We have released a full implementation of our network permutation method and the edge prior as an open-source Python package on GitHub and the Python Package Index.
+Important tasks in biomedical discovery, such as repurposing drugs or predicting the function of genes and their association to disease, are often framed as network edge prediction problems.
+In most real-world networks, the number of edges per node (called "degree") varies drastically.
+Since degree strongly influences edge prediction, any imbalance or bias in the distribution of degrees leads to nonspecific or misleading predictions.
```

Comment:
You haven't made the case that degree strongly influences edge prediction yet. It hasn't come up. You could say _If_...